addons/Popochiu/Engine/Popochiu.gd

```diff
@@ -62,8 +63,12 @@ onready var _saveload := SaveLoad.new()
 
 # ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░ GODOT ░░░░
 func _ready() -> void:
+	if Engine.editor_hint: return
```

Comment:
What are you doing here? Returning when you are running the editor I guess, but why?